bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -4,9 +4,10 @@
 
 {% from "macros.html" import monitor_button, fxa_cta_link with context %}
 
-{% add_lang_files "firefox/whatsnew_71" %}
```

Comment:
I'm assuming the lang file isn't needed because the experiment is EN only?

Reply:
Correct yeah